docs/release_notes/v1.7.3.md

```diff
@@ -0,0 +1,15 @@
+# Dapr 1.7.3
+
+### Fixes incorrect actor state-store configuration behavior
+
+#### Problem
+
+Omitting a state store configuration will cause the actor subsystem to fail to initialize. This is the correct behavior for services providing actors but this not for services acting as consumers / clients of actors that other services provide. A service acting purely as a consumer does not need state storage and should initiazlize correctly in order to invoke actors elsewhere. 
+
+#### Root cause
+
+Absence of a state store causes the actor subsystem to throw an error, regardless of whether or not the service is providing hosted actors.
```

Comment:
Root Cause: All Actor APIs return an error if no actor state store is defined, regardless of where actors are registered.

FYI SDKs use the term "register" for telling a sidecar to host particular actors. So this may be less confusing to use in language.